samples/subsys/mgmt/mcumgr/smp_svr/README.rst

```diff
@@ -220,6 +220,12 @@ device.
 If all goes well the image will now be stored in slot-1, ready to be swapped
 into slot-0 and executed.
 
+Note that at the beginning of upload process the target mights starts erasing
```

Comment:
Here's a replacement with some grammar and spelling fixes:

    .. note:: 

       At the beginning of the upload process, the target might start erasing
       the image slot, taking several dozen seconds for some targets.  This might
       cause an NMP timeout in the management protocol tool. Use the
       ``-t <timeout-in-seconds`` option to increase the response timeout for the
       ``mcumgr`` command line tool if this occurs.

Reply:
@dbkinder done